internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv

```diff
@@ -336,26 +320,22 @@ geoId/08033,Dolores
 geoId/08039,Elbert
 geoId/08043,Fremont
```

Comment:
remove "Fremont"?

Reply:
I tried the query that is fine matches city